notebooks/advanced_topics/IDC_Recipes_Conversion.ipynb

```diff
@@ -0,0 +1,2064 @@
+      "DicomReader initializing rgb-dicom/5ee2770f-b222-4565-a3bd-46ba294ca222.dcm\n",
+      "Verifying DICOM format\n",
+      "Reading tags\n",
+      "Calculating image offsets\n",
+      "Building file list\n",
+      "Populating metadata\n",
+      "Verifying DICOM format\n",
+      "Reading tags\n",
+      "Calculating image offsets\n",
+      "Building file list\n",
+      "Populating metadata\n",
+      "Verifying DICOM format\n",
+      "Reading tags\n",
+      "Calculating image offsets\n",
+      "Building file list\n",
+      "Populating metadata\n",
+      "Verifying DICOM format\n",
+      "Reading tags\n",
+      "Calculating image offsets\n",
+      "Building file list\n",
+      "Populating metadata\n",
+      "Verifying DICOM format\n",
+      "Reading tags\n",
+      "Calculating image offsets\n",
+      "Building file list\n",
+      "Populating metadata\n",
+      "[DICOM] -> rgb-dicom/output.ome.tiff [OME-TIFF]\n",
+      "Tile size = 512 x 512\n",
+      "\tConverted 1/1 planes (100%)\n",
+      "[done]\n",
+      "3.466s elapsed (66.0+321.0ms per plane, 2888ms overhead)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Convert DICOM to OME-TIFF without precompressed tiles\n",
+    "\n",
+    "# Convert using bfconvert with non-default options:\n",
+    "# Crop the full resolution image and drop the resolution pyramid because the output image is small\n",
+    "# A 1000x1000 region of the full resolution image starting at (5000,5000) will be converted.\n",
+    "\n",
+    "# Tile sizes do not need to be specified if the tile dimensions in the original DICOM should be preserved,\n",
+    "# but here tiles are explicitly set to 512x512 to demonstrate the '-tilex' and '-tiley' options.\n",
+    "\n",
+    "# If converting the entire input dataset instead:\n",
+    "# - omit the '-crop 5000,5000,1000,1000' option\n",
+    "# - replace the '-series 0' option with '-noflat'\n",
+    "\n",
+    "!./bftools/bfconvert -series 0 -compression JPEG-2000 -crop 5000,5000,1000,1000 -overwrite -tilex 512 -tiley 512 rgb-dicom/5ee2770f-b222-4565-a3bd-46ba294ca222.dcm rgb-dicom/output.ome.tiff\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "IR-wRs44E85C"
+   },
+   "source": [
+    "## Conversion using bioformats2raw, raw2ometiff and NGFF-Converter\n",
+    "\n",
+    "There are also several tools which make use of Bio-Formats, and can be used to convert to either [OME-NGFF/OME-Zarr](https://ngff.openmicroscopy.org/latest/index.html) or OME-TIFF. These include the [bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) command line tool, which converts any input format supported by Bio-Formats to OME-NGFF/OME-Zarr, and [raw2ometiff](https://github.com/glencoesoftware/raw2ometiff), which converts OME-NGFF/OME-Zarr to OME-TIFF."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "BSgOKNYErcGz"
+   },
+   "source": [
+    "### Install required packages"
```

Comment:
As in the other notebook, I would install all prerequisites in a dedicated cell in the beginning of the notebook.